app/controllers/internal/configs_controller.rb

```diff
@@ -131,7 +130,7 @@ def image_params
       logo_png
       logo_svg
       main_social_image
-      primary_sticker_image_url
+      secondary_logo_url
```

Comment:
If you look through the conversations on this PR (sorry there's a lot 😅) you'll see that we had a discussion about `primary_sticker_image_url`. We concluded that it was not an intuitive name, and that `secondary_logo` made more sense.